### PR TITLE
Migrate Request to the new design - Closes #757

### LINKED ISSUE #757: Migrate Request to the new design
This is a subtask of #746
The goal is to migrate request to the new design according to the designs here:
https://projects.invisionapp.com/d/main#/console/17093670/355136612/preview

### PR DESCRIPTION
**Pull request by colado**
# What was the bug or feature?
Described in #757.

### How did I fix it?
Implemented design changes on screen.

## Type of change
- Enhancement (a non-breaking change which adds functionality)

### How to test it?
Navigate to Request tab.

# Checklist:
- My code follows the style guidelines of this project
- I have performed a self-review of my own code
- I have commented my code, particularly in hard-to-understand areas
- I have made corresponding changes to the documentation
- I have added tests that prove my fix is effective or that my feature works
- New and existing unit tests pass locally with my changes

### CODE REVIEW

**Comment by colado** [comment]
@yasharAyari I fixed the scrollview issue here. Can be reviewed now.

File: src/components/request/index.js [hunk 1]
```diff
@@ -59,73 +61,90 @@ class Request extends React.Component {
...
-              </Share>
+              <View style={styles.addressContainer}>
+                { activeToken === tokenMap.LSK.key && (
+                  <Avatar style={styles.avatar} address={address} size={24} />
+                )}
+                <B style={[styles.address, styles.theme.address]}>
+                  {address}
+                </B>
+                <Icon
+                  onPress={this.copyToClipboard}
```
Comment (yasharAyari) [inline src/components/request/index.js]: You should use `CopyToClipBoard` component instead of using Clipboard method of react-native.

Comment (colado) [reply src/components/request/index.js]: I removed the Clipboard method and used `CopyToClipboard` component instead.

Comment (reyraa) [reply src/components/request/index.js]: Please remove the class `copyToClipboard` method and the native `Clipboard` object.

Comment (colado) [reply src/components/request/index.js]: Forgot to remove those. They're removed now.

File: src/components/request/index.js [hunk 2]
```diff
@@ -1,6 +1,6 @@
 import React from 'react';
 import connect from 'redux-connect-decorator';
-import { View, TouchableWithoutFeedback } from 'react-native';
+import { View, TouchableWithoutFeedback, Platform } from 'react-native';
```
Comment (reyraa) [inline src/components/request/index.js]: Prefer the `deviceType` method from the utilities.

File: src/components/request/index.js [hunk 3]
```diff
@@ -65,67 +67,78 @@ class Request extends React.Component {
...
     const { address } = account[activeToken];
+    const isAndroid = Platform.OS === 'android';
 
     return (
       <View style={[styles.wrapper, styles.theme.wrapper]}>
         <KeyboardAwareScrollView
           enableOnAndroid={true}
           enableResetScrollToCoords={false}
-          contentContainerStyle={styles.container}
+          extraHeight={isAndroid && 170}
```
Comment (reyraa) [inline src/components/request/index.js]: > Failed prop type: Invalid prop `extraHeight` of type `boolean` supplied to `KeyboardAwareScrollView`, expected `number`.

**Comment by reyraa** [comment]
There's also a slight visual issue. The copy to clipboard component has a reactive response message that breaks to second line:
Screenshot 2019-05-22 at 11 35 37

File: src/components/request/index.js (hunk 2 above)
Comment (colado) [reply src/components/request/index.js]: Replaced Platform with deviceType utility

File: src/components/request/index.js (hunk 3 above)
Comment (colado) [reply src/components/request/index.js]: I've refactored this and removed the first boolean from there.

**Comment by colado** [comment]
@reyraa I've also extended the width of `copyToClipboard`  so that it doesn't break to the second line.

**Pull request merged**